src/app/components/Menu.tsx

```diff
@@ -98,19 +98,17 @@ export default function Menu() {
     <nav className={styles.container}>
       {width > 500 ? (
         <a href="/#" className={styles.container__logo}>
-          <SvgTwitter />
+          <SvgLogo />
         </a>
       ) : null}
-      {menu.map(menuButton => {
-        const { name, image, link } = menuButton
-
+      {menu.map(({ name, image, link }) => {
         return (
```

Comment:
Here I meant to rewrite to something like that:
```tsx
{menu.map(({ name, image, link }) => (
  <MenuUnit
    key={name}
    name={name}
    image={image}
    link={link}
    active={activeButton === name}
    setActive={setActiveButton}
  />
)
...
```

But even better if it's like that:
```tsx
menu.map(menuUnitProps => (
  <MenuUnit 
    {...menuUnitProps} 
    key={menuUnitProps.name} 
    isActive={activeButton === menuUnitProps.name}
    setActive={setActiveButton}
  />
)
```